plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -103,6 +103,22 @@ public void testSystemJdbc()
                         "WHERE table_schem = 'tpch' AND table_name LIKE '%system_jdbc_test%'",
                 "VALUES ('iceberg', 'tpch', 'system_jdbc_test', 'TABLE')");
 
+        assertThat(query(
+                "SELECT table_cat, table_schem, table_name, column_name, type_name " +
+                        "FROM system.jdbc.columns " +
+                        "WHERE table_schem LIKE '%tpch%'"))
+                .skippingTypesCheck()
+                .containsAll(
+                        "VALUES ('iceberg', 'tpch', 'system_jdbc_test', '_bigint', 'bigint'), " +
+                                "('iceberg', 'tpch', 'system_jdbc_test', '_date', 'date')");
```

Comment:
let's add this in `BCT.testMaterializedView`, maybe without type names, for the sake of connectors which e.g. do not suppport bigint